plugins/aggregators/final/README.md

```diff
@@ -32,6 +32,9 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## aggregator and will not get sent to the output plugins.
   # drop_original = false
 
+  ## If true, _final is added to every field name
+  # rename_fields = true
+
```

Comment:
How about naming this `keep_original_field_names` with a default of `false`? This way, you don't need to initialize the variable or modify the tests...